internal/block/block_pool.go

```diff
@@ -89,6 +89,10 @@ func (bp *BlockPool) Get() (Block, error) {
 	}
 }
 
+func (bp *BlockPool) FreeBlocksChannel() chan Block {
```

Comment:
unit tests?

Reply:
Added an explicit test. This was indirectly tested earlier based on the codecov report.